samples/mpsl/timeslot/README.rst

```diff
@@ -18,7 +18,7 @@ The sample supports any one of the following development kits:
 
 .. note::
    For the nRF5340 DK, this sample is only supported on the network core (``nrf5340dk_nrf5340_cpunet``), and the :ref:`nrf5340_empty_app_core` sample must be programmed to the application core.
-
+   For the nRF54h20 DK, this sample is only supported on the radio core (``nrf5340dk_nrf5340_cpunet``) and (``nrf54h20dk_nrf4h20_cpurad``).
```

Comment:
Maybe you wanted to combine the sentences for 53 and 54H?
```suggestion
   For the nRF54h20 DK, this sample is only supported on the radio core (``nrf54h20dk_nrf4h20_cpurad``).
```

Reply:
```suggestion
   For the nRF54H20 DK, this sample is only supported on the radio core (``nrf54h20dk_nrf4h20_cpurad``).
```
Does the nRF54H20 DK need an empty Application Core application like the nRF5340 DK does?

Also, you need to add an empty line after this one to fix the doc build.

Reply:
No there is no need for any FW to be run on APP core. This is why I went with separate line for 54H to avoid complex sentence.